src/ResourceManagement/Authorization/Authorization/Generated/AuthorizationManagementClient.cs

```diff
@@ -31,7 +31,8 @@ namespace Microsoft.Azure.Management.Authorization
     public partial class AuthorizationManagementClient : ServiceClient<AuthorizationManagementClient>, IAuthorizationManagementClient
     {
         private string _apiVersion;
-        
+        public const string APIVersion = "2015-07-01";
```

Comment:
Sorry, maybe I misunderstood your commit. The code is generated by Hydra. I changed this for code maintenance, or I can leave as it is. We do not change the Hydra spec.